### ISSUE TITLE
Typo in 'Read this page in' markup

### ISSUE THREAD

**Issue by gosko** [w3c/w3c-website#697]
The markup around 'Read this page in', for example on the [Standards overview page](https://www.w3.org/standards/) has a typo:
```html
<span><span lange="en">Read this page in:</span></span>
```
(`lange` instead of `lang`)

Also, is this `span` actually necessary? It's not present in the other translation files nearby, and I expect it's only needed when there is text written in a language other than the page's primary language. (but I'm just guessing)

Also I wonder if the other `span` is needed as well.

**Comment by vivienlacourba** [comment]
> The markup around 'Read this page in', for example on the [Standards overview page](https://www.w3.org/standards/) has a typo:
> 
> ```html
> <span><span lange="en">Read this page in:</span></span>
> ```
> 
> (`lange` instead of `lang`)
> 
> Also, is this `span` actually necessary? It's not present in the other translation files nearby, and I expect it's only needed when there is text written in a language other than the page's primary language. (but I'm just guessing)

I agree with @gosko the `<span lang="en">` does not seem necessary if it matches the current's page language.

> Also I wonder if the other `span` is needed as well.

This `<span>` is present on https://design-system.w3.org/components/translations.html and indeed seem superfluous at first glance. But Removing it affects the display on the landing template while it doesn't on the default template (tested with Firefox).

On the landing template, removing it removes some spacing between "Read the page in:" and the list of languages when those are on the same line e.g. on https://www.w3.org/standards/about/

* Page using the 'landing' template page e.g.  https://www.w3.org/standards/
With the surrounding `<span>` around "Read this page in:"
![image](https://github.com/user-attachments/assets/9d087c7a-8316-48a0-abd2-9342d07d8e4a)
Without it:
![image](https://github.com/user-attachments/assets/ad7f4626-a92f-4dd9-80ec-6981720dfe39)

* Page using the 'default' template e.g.  https://www.w3.org/standards/about/
With the surrounding `<span>` around "Read this page in:"
![image](https://github.com/user-attachments/assets/01cd6b48-5a01-4b37-980d-d6b9c18dce0a)
Without it:
![image](https://github.com/user-attachments/assets/b7f7a0c3-521f-4f38-9f7a-047f51daad6c)

**Comment by jean-gui** [comment]
I have fixed the typo. The attribute is indeed not necessary browsing the English version, but when you read a page in a language for which we don't have the full translations, https://www.w3.org/fr/press-releases/2023/w3c-welcomes-seth-dobbs-as-new-ceo/ for example, it then becomes necessary. As knowing when it is needed or not can be a tricky, I felt it was better to always have it as it does no harm.
